lib/vm/src/global.rs

```diff
@@ -6,10 +6,9 @@ use thiserror::Error;
 use wasmer_types::{GlobalType, Mutability, Type, Value};
 
 #[derive(Debug)]
-/// TODO: figure out a decent name for this thing
```

Comment:
```suggestion
/// A Global instance
```

Reply:
github doesn't let me commit suggestions that replace deleted lines. :(